macros/WebExtAPIOverview.ejs

```diff
@@ -2,22 +2,27 @@
 
 var output = "";
 var apiPages;
-var apiSummaryId = "api-summary";
 
-if ($0) {
-  apiPages = page.subpagesExpand($0);
-}
-else {
-  apiPages = page.subpagesExpand();
-}
+apiPages = page.subpagesExpand('/en-US/Add-ons/WebExtensions/API');
+
+var apiLinkLabel = mdn.localString({
+    "en-US": "API reference documentation",
+    "zh-CN": "API 参考资料"
+});
 
 for (var i=0; i < apiPages.length; i++) {
   var apiPage = apiPages[i];
   var apiPageUrlPieces = apiPage.url.split("/");
   var apiName = apiPageUrlPieces[apiPageUrlPieces.length-1];
+  var localepath = '/' + env.locale + '/docs/Mozilla/Add-ons/WebExtensions/API/' + apiName;
+  if (apiPage.url) {
+    var apiPage = wiki.getPage(localepath);
+    var apiPageUrlPieces = apiPage.url.split("/");
```

Comment:
And, if ``apiPage`` is empty, ``apiPage.url`` will be undefined, and you'll get an error.